spec/rails_helper.rb

```diff
@@ -4,6 +4,9 @@
 abort('DATABASE_URL environment variable is set') if ENV['DATABASE_URL']
 
 require 'rspec/rails'
+require 'webmock/rspec'
+require 'vcr'
+require 'database_cleaner'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.